### DISCUSSION TITLE
Is it possible to preview images prior to Import?

### DISCUSSION THREAD

**Discussion by Godd67** [photoprism/photoprism#3195] [Q&A]
My phone photos are synced to Photoprism Import folder automatically and then I import them like once a week. I would like to preview them before importing into originals to filter out garbage photos or photos that should not be imported at all :)
Is it possible to achieve without download them first to PC, checking what are they , then delete them from Import folder and then click Import?

**Comment by graciousgrey** [comment]
That's not possible a the moment. In order to be able to display images in PhotoPrism they need to be indexed first, which currently  happens after they have been moved to Originals.

This feature request seems to be related to your problem: https://github.com/photoprism/photoprism/issues/943
